sections/sprint-1/networking.tex

```diff
@@ -263,3 +263,62 @@ \subsection{IPv4 and IPv6}\label{ipv4-ipv6}
 IPv6 addresses are usually written as eight 16-bit hexadecimal numbers \cite{socketnetworking}.
 The higher-order bits of the 128-address imply the type.
 For this project, this should not have a significant impact, and either can be used.
+
+
+\subsection{Format of data}
+As it is needed to send data over the network for our game, it is required to define the format it is sent as.
+The network format for the field anchor, player position and goal position is \texttt{0xyyyyxxxxiisstt}, where \texttt{t} is the type, \texttt{s} is the timestamp, \texttt{i} is the ID, \texttt{xxxx} is the \texttt{x} position and \texttt{yyyy} is the \texttt{y} position.
+These formats can be seen on \autoref{Tab:anchor-format}, \autoref{Tab:player-format} and \autoref{Tab:player-format}.
+These are almost identical, as the same information is required.
+The only difference between them is the type, and the context of the IDs.
+The type is at the end of the hexadecimal, so that it is easy to bitshift, and then switch on which type it is.
```

Comment:
```suggestion
The type is at the end of the hexadecimal so that it is easy to bitshift, and then switch on which type it is.
```

er ikke stor fan af den her sætning, det virker som et ret halvt argument